### ISSUE TITLE
File not found: Wrong resolving of dependency imports.

### ISSUE THREAD

**Issue by delijah** [css-modules/css-modules#171]
Let's assume we have the following file structure:

```
+
|+modules
||-magicButton.css
|+shared
||-layout.css
||-typography.css
```

File contents look like this:

```
/* modules/magicButton.css */
.title {
  composes: app-title from "./../shared/layout.css";
}
```

```
/* shared/layout.css */
.app-title {
  composes: large-font from "./typography.css";
}
```

```
/* shared/typography.css */
.large-font {
  font-size: 20px;
}
```

When running this, i would get a `Module not found: Error: Cannot resolve 'file' or 'directory'`.

When i use `"./../shared/typography.css"` instead of `"./typography.css"` in the "shared/layout.css" file, everything works fine. This is wrong!

Related to #170.

**Comment by delijah** [comment]
Hey. Any progress on this? @geelen @StevenIseki @sokra ?

**Comment by delijah** [comment]
To me it looks like this has been resolved (checked with css-loader 0.28.0 and webpack 2.5.1).
